### ISSUE TITLE
[Bug]: Sidebar gets cut when a sidebar item name is too long

### ISSUE THREAD

**Issue by talkor** [storybookjs/storybook#25094]
### Describe the bug

When a page/story name is too long the entire sidebar is cut when you resize it to the minimum. See image below:
<img width="398" alt="Screenshot 2023-12-03 at 16 37 14" src="https://github.com/storybookjs/storybook/assets/18269880/600cd476-4374-494a-9e16-97d49ca83e8d">

### To Reproduce

- Add a long title to any page's Title
- Resize the sidebar to minimum
Result:
Sidebar is cut

Link to reproduce: https://stackblitz.com/edit/github-w6pgws?file=src%2Fstories%2FConfigure.mdx&preset=node

### System

```bash
Storybook Environment Info:

  System:
    OS: macOS 14.0
    CPU: (12) arm64 Apple M2 Pro
    Shell: 3.2.57 - /bin/sh
  Binaries:
    Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
    Yarn: 1.22.19 - /opt/homebrew/bin/yarn
    npm: 8.5.0 - ~/.nvm/versions/node/v16.14.2/bin/npm <----- active
  Browsers:
    Chrome: 119.0.6045.199
    Safari: 17.0
  npmPackages:
    @storybook/addon-a11y: ^7.5.3 => 7.5.3 
    @storybook/addon-actions: ^7.5.3 => 7.5.3 
    @storybook/addon-controls: ^7.5.3 => 7.5.3 
    @storybook/addon-docs: ^7.5.3 => 7.5.3 
    @storybook/addon-essentials: ^7.5.3 => 7.5.3 
    @storybook/addon-interactions: ^7.5.3 => 7.5.3 
    @storybook/addon-links: ^7.5.3 => 7.5.3 
    @storybook/addon-storyshots: ^7.5.3 => 7.5.3 
    @storybook/addon-styling-webpack: ^0.0.5 => 0.0.5 
    @storybook/addon-themes: ^7.5.3 => 7.5.3 
    @storybook/addon-toolbars: ^7.5.3 => 7.5.3 
    @storybook/addons: ^7.5.3 => 7.5.3 
    @storybook/blocks: ^7.5.3 => 7.5.3 
    @storybook/jest: ^0.2.3 => 0.2.3 
    @storybook/manager-api: ^7.5.3 => 7.5.3 
    @storybook/preset-scss: ^1.0.3 => 1.0.3 
    @storybook/react: ^7.5.3 => 7.5.3 
    @storybook/react-webpack5: ^7.5.3 => 7.5.3 
    @storybook/source-loader: ^7.5.3 => 7.5.3 
    @storybook/storybook-deployer: ^2.8.16 => 2.8.16 
    @storybook/testing-library: ^0.2.2 => 0.2.2 
    @storybook/theming: ^7.5.3 => 7.5.3 
    chromatic: ^7.5.3 => 7.6.0 
    eslint-plugin-storybook: ^0.6.15 => 0.6.15 
    storybook: ^7.5.3 => 7.5.3
```

### Additional context

Probably happens because of [Radix's Scroll Area](https://www.radix-ui.com/primitives/docs/components/scroll-area), they add a `display: table` as inline style to this element which is being used in storybook's sidebar. Removing this property fixes the issue
<img width="448" alt="Screenshot 2023-12-03 at 16 37 28" src="https://github.com/storybookjs/storybook/assets/18269880/97f9b52c-49d4-4cb5-a63f-91f3f0f27578">

**Comment by cdedreuille** [comment]
This should be fixed in 8.0. I'll see if we can find some time to fix it for 7.6 as well.

**Comment by shilman** [comment]
To try out the latest 8.0 prerelease:

```
npx storybook@next upgrade --prerelease
```
